# Examen 1

## Situación 1: Triángulo de Sierpinski

**1. ¿Cuál es el área de un triángulo equilátero de lado $a$?**

**(2 puntos)**

<img src="sierpinski1.png" align="middle" width="300">

A. $\dfrac{1}{2}a^2$

B. $\dfrac{3}{4}a^2$

C. $\dfrac{\sqrt{3}}{2}a^2$

D. $\dfrac{\sqrt{3}}{4}a^2$

El área de este triángulo se puede calcular mendiante la fórmula del área para cualquier triángulo:

$$ A=\dfrac{(\text{cateto 1})\cdot(\text{cateto 2})\cdot\sin\,(\text{ángulo entre ambos catetos})}{2} $$

Pero, como se trata de un triángulo, todo los lados valen $a$ y todos los ángulos valen $60^\circ$.

$$ A_\Delta=\dfrac{a^2\sin60^\circ}{2} $$

In [1]:
sind(60)/2

0.4330127018922193

Entonces, podemos calcular $\dfrac{\sin 60^\circ}{2}=0.433\dots$ con los coeficientes de las opciones:

Para A. el coeficiente es $\dfrac{1}{2}=0.5$

Para B. el coeficiente es $\dfrac{3}{4}=0.75$

Para C. el coeficiente es $\dfrac{\sqrt{3}}{2}=0.866\dots$

In [2]:
(√3)/2

0.8660254037844386

Para D. el coeficiente es $\dfrac{\sqrt{3}}{4}=0.433\dots$

In [3]:
(√3)/4

0.4330127018922193

Entonces el área del triángulo equilátero es

$$ A_\Delta=\dfrac{\sqrt{3}}{4}a^2 $$

A. $\dfrac{1}{2}a^2$

B. $\dfrac{3}{4}a^2$

C. $\dfrac{\sqrt{3}}{2}a^2$

D. $\dfrac{\sqrt{3}}{4}a^2$

**Solución:** D.

**2. El triángulo de Sierpinski es un fractal que se puede construir a partir de cualquier triángulo. Su primera fase se construye calculando el punto medio de todos sus lados de manera que podamos encontrar cuatro triángulos semejantes que caben perfectamente dentro del triángulo original. Al construir la primera fase del triángulo de Sierpinski con un triángulo equilátero de lado $a$, ¿con qué expresión calculamos el área sombreada?**

**(2 puntos)**

<img src="sierpinski2.png" align="middle" width="300">

A. $\dfrac{3\sqrt{3}}{4}a^2$

B. $\dfrac{9}{4}a^2$

C. $\dfrac{3\sqrt{3}}{16}a^2$

D. $\dfrac{3}{16}a^2$

Como dividimos el triángulo en cuatro partes y tomamos tres, el área de la primera fase del triángulo de Sierpinski es

$$ \dfrac{3}{4}A_\Delta=\dfrac{3}{4}\cdot\dfrac{\sqrt{3}}{4}a^2=\dfrac{3\sqrt{3}}{16}a^2 $$

A. $\dfrac{3\sqrt{3}}{4}a^2$

B. $\dfrac{9}{4}a^2$

C. $\dfrac{3\sqrt{3}}{16}a^2$

D. $\dfrac{3}{16}a^2$

**Solución:** C.

**3.  Al tratarse de un fractal, se puede repetir el proceso de la primera fase tantas veces como queramos. ¿Cuál es la proporción entre el área sombreada de la primera fase y la segunda fase (en la figura adjunta)?**

**(3 puntos)**

<img src="sierpinski3.png" align="middle" width="300">

A. $\dfrac{3}{2}$

B. $\dfrac{4}{3}$

C. $\dfrac{4}{5}$

D. $\dfrac{9}{16}$

Ya vimos que el área de la primera fase es

$$ \dfrac{3}{4}A_\Delta $$

Notemos que para la segunda fase es como si separaramos el triángulo original en dieciséis partes y tomamos nueve. Entonces el área de la segunda fase es

$$ \dfrac{9}{16}A_\Delta $$

Finalmente, la proporción de la primera fase respecto de la segunda es

$$ \dfrac{\dfrac{3}{4}A_\Delta}{\dfrac{9}{16}A_\Delta}=\dfrac{3\cdot16}{9\cdot4}=\dfrac{4}{3} $$

A. $\dfrac{3}{2}$

B. $\dfrac{4}{3}$

C. $\dfrac{4}{5}$

D. $\dfrac{9}{16}$

**Solución:** B.

**4. Suponiendo que el triángulo original tiene 10 pulgadas (in en inglés) de lado, ¿cuál es el área sombreada de la tercera fase?**

**(3 puntos)**

<img src="sierpinski4.png" align="middle" width="300">

A. $\dfrac{675\sqrt{3}}{64}\ \text{in}^2$

B. $\dfrac{675\sqrt{3}}{16}\ \text{in}^2$

C. $\dfrac{365\sqrt{3}}{64}\ \text{in}^2$

D. $\dfrac{365\sqrt{3}}{16}\ \text{in}^2$

Para este punto podemos notar que cada fase representa tres cuartas partes de la anterior, por lo que, para la tercera fase el area será

$$ \left(\dfrac{3}{4}\right)^3A_\Delta=\dfrac{3^3}{4^3}A_\Delta=\dfrac{27}{64}A_\Delta=\dfrac{27}{64}\cdot\dfrac{\sqrt{3}}{4}a^2=\dfrac{27\sqrt{3}}{256}a^2 $$

Si $a=10\ \text{in}$, el área de la tercera fase es

$$ \dfrac{27\sqrt{3}}{256}a^2=\dfrac{27\sqrt{3}}{256}(10\ \text{in})^2=\dfrac{2700\sqrt{3}}{256}\ \text{in}^2=\dfrac{675\sqrt{3}}{64}\ \text{in}^2 $$

A. $\dfrac{675\sqrt{3}}{64}\ \text{in}^2$

B. $\dfrac{675\sqrt{3}}{16}\ \text{in}^2$

C. $\dfrac{365\sqrt{3}}{64}\ \text{in}^2$

D. $\dfrac{365\sqrt{3}}{16}\ \text{in}^2$

**Solución:** A.

## Situación 2: Área y perímetro

**5. La figura muestra tres cuadrados consecutivos de $5$, $4$, y $3$ pulgadas (in) de lado, respectivamente. Encuentra el área de la región negra.**

**(2 puntos)**

<img src="negro.png" align="middle" width="550">

A. $40\ \text{in}^2$

B. $20\ \text{in}^2$

C. $30\ \text{in}^2$

D. $50\ \text{in}^2$

La manera más fácil de hacer esto es usando el triángulo blanco de base 12 y altura 5, y quitarle lo que le falta a la región negra.

Para que el triángulo blanco se vuelva el negro, necesitamos quitarle dos rectángulos: uno de 4 por 1, y otro de 3 por 2.

Entonces

$$ A=A_\text{triangulo}-A_\text{rectangulo 1}-A_\text{rectangulo 2}=\dfrac{12\cdot5}{2}-4\cdot1-3\cdot2=20 $$

**Solución:** B.

**6. Encuentra el perímetro de la misma región negra.**

**(2 puntos)**

A. $48\ \text{in}$

B. $13\ \text{in}$

C. $30\ \text{in}$

D. $37\ \text{in}$

Primero obtenemos el valor de la diagonal. Esto se puede hacer muy simple con Pitágoras.

$$ \text{(c.o.)}^2+\text{(c.a.)}^2=\text{(hip.)}^2 $$

Entonces

$$ \text{hip.}=\sqrt{\text{(c.o.)}^2+\text{(c.a.)}^2}=\sqrt{(5)^2+(12)^2}=\sqrt{25+144}=\sqrt{169}=13 $$

Luego, yendo de izquierda a derecha por arriba y luego de derecha a izquierda por abajo, el perimetro es (ya no fui más específico con esto porque me habría tardado más haciendo imágenes al respecto pero ojalá puedan entenderle y, si no, pueden preguntarme en la clase)

$$ P=5+1+4+1+3+3+13=30 $$

A. $48\ \text{in}$

B. $13\ \text{in}$

C. $30\ \text{in}$

D. $37\ \text{in}$

**Solución:** C.

**7. La figura muestra dos circunferencias dentro de un rectángulo de 10 por 20 pulgadas (in). Encuentra el área de la región roja.**

**(2 puntos)**

<img src="rojo.png" align="middle" width="350">

A. $(200-100\pi)\ \text{in}^2$

B. $(100-25\pi)\ \text{in}^2$

C. $(40+10\pi)\ \text{in}^2$

D. $(150+20\pi)\ \text{in}^2$

Veamos que el área de la región roja es la mitad de la diferencia del área del rectángulo y el área de los dos círculos de radio $5\ \text{in}$. Entonces,

$$ A=\dfrac{A_\text{rectangulo}-A_\text{2 circunferencias}}{2}=\dfrac{(20\cdot10)-(\pi\cdot5^2+\pi\cdot5^2)}{2}=100-25\pi $$

A. $(200-100\pi)\ \text{in}^2$

B. $(100-25\pi)\ \text{in}^2$

C. $(40+10\pi)\ \text{in}^2$

D. $(150+20\pi)\ \text{in}^2$

**Solución:** B.

**8. Calcula la longitud de la cuerda azul en la imagen, redondeada a dos décimas. *Pista: Nos falta el ángulo de la cuerda respecto al centro del círculo. Para encontrarlo, dibuja una recta paralela a la base del rectángulo y que pase por el punto medio de su altura.***

**(3 puntos)**

<img src="rojo_cuerda.png" align="middle" width="350">

A. $17.10\ \text{in}$

B. $14.08\ \text{in}$

C. $8.94\ \text{in}$

D. $9.67\ \text{in}$

Notemos que hay un triángulo rectángulo de altura 10 y base 20. El ángulo que se forma a la izquierda de éste es

In [4]:
atand(10/20)

26.56505117707799

Notemos que el triángulo que se forma en el centro del círculo es un triángulo oblicuángulo cuyos lados son dos radios y el otro lado es la cuerda.

<img src="cuerda_sol.png" align="middle" width="500">

Entonces, el ángulo que se forma en el centro del círculo es

In [5]:
180-2*26.56505117707799

126.86989764584402

Podemos usar ley de cosenos para calcular la longitud de la cuerda:

$$ \ell^2=R^2+R^2-2R^2\cos\theta $$

$$ \ell=\sqrt{2R^2(1-\cos\theta)} $$

In [6]:
√(2(5)^2*(1-cosd(126.86989764584402)))

8.94427190999916

A. $17.10\ \text{in}$

B. $14.08\ \text{in}$

C. $8.94\ \text{in}$

D. $9.67\ \text{in}$

**Solución:** C.

**9. Encuentra el perímetro de la región roja redondeada a dos décimas.**

**(3 puntos)**

A. $65.89\ \text{in}$

B. $70.41\ \text{in}$

C. $78.45\ \text{in}$

D. $80.71\ \text{in}$

Veamos que el perímetro de la región roja es la mitad de la suma del perímetro del rectángulo y el perímetro de los dos círculos de radio 5 y la diagonal del rectángulo menos las cuerdas de los círculos.

$$ \dfrac{P_\text{rectangulo}+P_\text{2 circunferencias}}{2}=\dfrac{(20+10+20+10)+(2\pi\cdot5+2\pi\cdot5)}{2}=30+10\pi $$

Notemos que la diagonal mide

$$ \sqrt{10^2+20^2}=\sqrt{500} $$

por lo que, quitándole las dos cuerdas que se forman, obtenemos

In [7]:
(√500)-2*8.94427190999916

4.47213595499958

<img src="rojo.png" align="middle" width="350">

In [8]:
30+10π+4.47213595499958

65.88806249089751

A. $65.89\ \text{in}$

B. $70.41\ \text{in}$

C. $78.45\ \text{in}$

D. $80.71\ \text{in}$

**Solución:** A.

## Situación 3: Cisterna

**Una cisterna tiene 2.4 m de ancho y una persona se sitúa a 1.3 m del borde. Desde una altura de 1.8 m (donde se encuentran sus ojos), la persona observa que el borde de la cisterna coincide con la
línea del fondo como se muestra en la figura.**

<img src="cisterna_profunda.png" align="middle" width="350">

**10. ¿Qué profundidad tiene la cisterna? Redondea el valor a dos décimas.**

**(2 puntos)**

A. $4.02\ \text{m}$

B. $3.78\ \text{m}$

C. $3.32\ \text{m}$

D. $4.64\ \text{m}$

Podemos identificar esto con triángulos semejantes y usar Tales:

$$ \dfrac{\text{profundidad de la cisterna}}{\text{altura de los ojos de la persona}}=\dfrac{\text{ancho de la cisterna}}{\text{distancia de la persona a la orilla}} $$

$$ \text{profundidad de la cisterna}=\dfrac{2.4}{1.3}\cdot1.8\approx3.32 $$

In [9]:
2.4*1.8/1.3

3.3230769230769233

A. $4.02\ \text{m}$

B. $3.78\ \text{m}$

C. $3.32\ \text{m}$

D. $4.64\ \text{m}$

**Solución:** C.

**11. ¿A qué distancia debe situarse la persona de la orilla, para que al observar el borde de la cisterna coincida justo con la mitad de su profundidad?**

**(2 puntos)**

A. $1\ \text{m}$

B. $2.2\ \text{m}$

C. $3.1\ \text{m}$

D. $2.6\ \text{m}$

Podemos identificar esto con triángulos semejantes y usar Tales:

$$ \dfrac{\text{distancia de la persona a la orilla}}{\text{ancho de la cisterna}}=\dfrac{\text{altura de los ojos de la persona}}{\text{mitad de la profundidad de la cisterna}} $$

$$ \text{distancia de la persona a la orilla}=\dfrac{1.8}{1.66}\cdot2.4 $$

In [10]:
3.3230769230769233/2

1.6615384615384616

In [11]:
1.8*2.4/1.6615384615384616

2.6

A. $1\ \text{m}$

B. $2.2\ \text{m}$

C. $3.1\ \text{m}$

D. $2.6\ \text{m}$

**Solución:** D.

**12. Después de observar la mitad de la profundidad, ¿qué distancia a la izquierda debe recorrer la persona desde el punto donde se encuentra para que al observar el borde de la cisterna coincida justo con la mitad del fondo de la cisterna (punto B)?**

**(3 puntos)**

A. $1.95\ \text{m}$

B. $2.15\ \text{m}$

C. $0.84\ \text{m}$

D. $2.56\ \text{m}$

$$ \dfrac{\text{distancia de la persona a la orilla}}{\text{mitad del ancho de la cisterna}}=\dfrac{\text{altura de los ojos de la persona}}{\text{profundidad de la cisterna}} $$

$$ \text{distancia de la persona a la orilla}=\dfrac{1.8}{3.32}\cdot1.2 $$

In [12]:
1.8*1.2/3.3230769230769233

0.65

Entonces, la diferencia de esta cantidad y la del ejercicio anterior es

In [13]:
2.6-0.65

1.9500000000000002

A. $1.95\ \text{m}$

B. $2.15\ \text{m}$

C. $0.84\ \text{m}$

D. $2.56\ \text{m}$

**Solución:** A.

| Problema | Respuesta | Puntaje | Problema | Respuesta | Puntaje |
|----------|-----------|---------|----------|-----------|---------|
| 1        | D         | 2       | 7        | B         | 2       |
| 2        | C         | 2       | 8        | C         | 3       |
| 3        | B         | 3       | 9        | A         | 3       |
| 4        | A         | 3       | 10       | C         | 2       |
| 5        | B         | 2       | 11       | D         | 2       |
| 6        | C         | 2       | 12       | A         | 3       |
 